bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vh012j_5f_NLO_FXFX_M_VToAll/vh012j_5f_NLO_FXFX_M_VToAll_madspin_card.dat

```diff
@@ -0,0 +1,13 @@
+set ms_dir ./madspingrid
+
+set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
```

Comment:
please use Nevents_for_max_weight instead

Reply:
ok, I will try it. Thanks for pointing out.

Reply:
Dear Meng,
We have updated the cards, please review them again.